### DISCUSSION TITLE
[M-Z] (12 Febbraio 2019) Esercizio 6

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#442] [Esami & Esoneri M-Z]
## Traccia

<img width="604" alt="image" src="https://github.com/user-attachments/assets/47f9766f-ab41-4dd2-9f1d-0b9d9fa03fc2">

**Comment by rimaout** [comment]
<img width="805" alt="image" src="https://github.com/user-attachments/assets/5f2558cf-e22e-4b35-aa1f-c80f60e69fe5">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190212.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190212.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/7de01af7-ec40-4d2f-be13-519811f88945)
